doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1457,6 +1496,21 @@ Abbreviations
    exception, if the right-hand side is just of the form :n:`@@qualid`,
    this conventionally stops the inheritance of implicit arguments.
 
+   Like for notations, it is possible to bind binders in
+   abbreviations. Here is an example:
+
+   .. coqtop:: none
+
+      Reset Initial.
+
+   .. coqtop:: in
```

Comment:
```suggestion
   .. coqtop:: in reset
```

Reply:
rest = reset ?? I would not have guessed!

Reply:
Thanks for the feedback and suggestions!